README.md

```diff
@@ -50,11 +50,13 @@ To deploy the app, you will need an Okta org setup as described in the section a
 You will also need to replace the config values for `ISSUER` and `CLIENT_ID` in Heroku configuration based on your Okta org.
 Also be sure to add your Heroku app's base URL to the list of trusted origins in your Okta admin settings.
 
-After you deploy the app, click on **View** on the result screen to navigate to the newly deployed app.
+After you deploy the app, you will also need to go to your Okta account to [enable cors](https://developer.okta.com/docs/guides/enable-cors/granting-cors/) and configure the `login` and `logout` redirect URI for your app's domain if they are not preconfigured for the newly created heroku domain. 
 
-Click **Login** button on the app. Browser displays the sign-in page to authenticate. 
+**Note:** Base Url, like `/okta-hosted-login` or `/custom-login`, will be needed for the redirect uris.
```

Comment:
Can we use `https://<myapp.heroku.com>/okta-hosted-login` `https://<myapp.heroku.com>/custom-login` as example instead?

Reply:
Also mention that will be needed for the redirect uris in your client application settings. i.e. add `in your client application settings`